Gems/Terrain/Code/Source/TerrainRenderer/ClipmapBounds.h

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/base.h>
+#include <AzCore/Math/Aabb.h>
+#include <AzCore/Math/Vector2.h>
+#include <AzCore/std/containers/vector.h>
+#include <TerrainRenderer/Vector2i.h>
+#include <TerrainRenderer/Aabb2i.h>
+
+namespace Terrain
+{
+
+    struct ClipmapBoundsDescritor
+    {
+        //! Size of the clipmap in width / height.
+        uint32_t m_size = 1024;
+
+        //! Current center location of the clipmap
+        AZ::Vector2 m_center = AZ::Vector2::CreateZero();
+
+        //! Unused area around edges of the clipmap. Center point must be moved by
+        //! at least this much before an update is triggered. This allows for larger
+        //! but less frequent updates.
+        uint32_t m_margin = 4;
+
+        //! Scale of the clip map compared to the world. A scale of 0.5 means that
+        //! a clipmap of size 1024 would cover 512 meters.
+        float m_scale = 1.0f;
+    };
+
+    struct ClipmapBoundsRegion
+    {
+        //! The world bounds of the updated region. Z is ignored.
+        AZ::Aabb m_worldAabb;
+
+        //! The clipmaps bounds of the updated region. Will always be between 0 and size.
+        //! Min inclusive, max exclusive. i.e., a single pixel would be min [0, 0], max [1, 1].
```

Comment:
David pointed this out as well, I wasn't familiar with this notation so I have to assume a fair number of readers won't be either. I removed the brackets entirely so as not to confuse this with interval notation. The comment itself should make sense since it says in English Min inclusive, max exclusive, and provides an example.